diff --git a/node_modules/@metaplex-foundation/js/dist/esm/plugins/candyMachineV2Module/asserts.mjs b/node_modules/@metaplex-foundation/js/dist/esm/plugins/candyMachineV2Module/asserts.mjs
index 591c0fe..82d1cdb 100644
--- a/node_modules/@metaplex-foundation/js/dist/esm/plugins/candyMachineV2Module/asserts.mjs
+++ b/node_modules/@metaplex-foundation/js/dist/esm/plugins/candyMachineV2Module/asserts.mjs
@@ -47,7 +47,7 @@ const assertCandyMachineV2IsLive = candyMachine => {
 
   const liveDate = candyMachine.goLiveDate;
 
-  if (!liveDate || liveDate.gte(now())) {
+  if (!liveDate || new Date(liveDate.toNumber()) > new Date(Date.now())) {
     throw new CandyMachineV2NotLiveError(liveDate);
   }
 };
